packages/components/src/base-control/index.js

```diff
@@ -8,6 +8,20 @@ import classnames from 'classnames';
  */
 import VisuallyHidden from '../visually-hidden';
 
+/**
+ * @typedef Props
+ * @property {string} id ID
+ * @property {import('react').ReactNode} help Help text
+ * @property {import('react').ReactNode} label Help text
```

Comment:
Should be Label…

It's tricky to name `ReactNode` properties because they can be React elements, strings…

Maybe Label content?